snapshots/testsuite/issues.go

```diff
@@ -114,23 +114,33 @@ func checkChown(ctx context.Context, t *testing.T, sn snapshots.Snapshotter, wor
 
 // checkRename
 // https://github.com/docker/docker/issues/25409
-func checkRename(ctx context.Context, t *testing.T, sn snapshots.Snapshotter, work string) {
-	t.Skip("rename test still fails on some kernels with overlay")
-	l1Init := fstest.Apply(
-		fstest.CreateDir("/dir1", 0700),
-		fstest.CreateDir("/somefiles", 0700),
-		fstest.CreateFile("/somefiles/f1", []byte("was here first!"), 0644),
-		fstest.CreateFile("/somefiles/f2", []byte("nothing interesting"), 0644),
-	)
-	l2Init := fstest.Apply(
-		fstest.Rename("/dir1", "/dir2"),
-		fstest.CreateFile("/somefiles/f1-overwrite", []byte("new content 1"), 0644),
-		fstest.Rename("/somefiles/f1-overwrite", "/somefiles/f1"),
-		fstest.Rename("/somefiles/f2", "/somefiles/f3"),
-	)
+func checkRename(ss string) func(ctx context.Context, t *testing.T, sn snapshots.Snapshotter, work string) {
+	return func(ctx context.Context, t *testing.T, sn snapshots.Snapshotter, work string) {
+		l1Init := fstest.Apply(
+			fstest.CreateDir("/dir1", 0700),
+			fstest.CreateDir("/somefiles", 0700),
+			fstest.CreateFile("/somefiles/f1", []byte("was here first!"), 0644),
+			fstest.CreateFile("/somefiles/f2", []byte("nothing interesting"), 0644),
+		)
+
+		var applier []fstest.Applier
+		if ss != "overlayfs" && ss != "Overlay" {
```

Comment:
Where is `Overlay` used?

Reply:
It is coming from https://github.com/containerd/containerd/blob/8f7cfbda29299c21b02955677a438c7542252201/snapshots/overlay/overlay_test.go#L60, but I cannot recall "overlayfs" case. Let me check.

Reply:
Updated to only use "overlayfs".